testing/citest/tests/kube_smoke_test.py

```diff
@@ -530,6 +848,31 @@ def test_d_create_find_image_pipeline(self):
   def test_e_run_find_image_pipeline(self):
     self.run_test_case(self.scenario.run_find_image_pipeline())
 
+  def test_e_1_save_statefulset_pipeline(self):
+    self.run_test_case(self.scenario.save_statefulset_pipeline())
+
+  def test_e_2_create_statefulset_pipeline(self):
+    self.run_test_case(self.scenario.create_statefulset_pipeline())
+
+  def test_f_1_save_daemonset_pipeline(self):
+    self.run_test_case(self.scenario.save_daemonset_pipeline())
+
+  def test_f_2_create_daemonset_pipeline(self):
+    self.run_test_case(self.scenario.create_daemonset_pipeline())
+
+  def test_g_1_save_delete_daemonset_pipeline(self):
+    self.run_test_case(self.scenario.save_delete_daemonset_pipeline())
+
+  def test_g_2_execute_delete_daemonset_pipeline(self):
+    self.run_test_case(self.scenario.execute_delete_daemonset_pipeline())
+
+  def test_g_1_save_delete_statefulset_pipeline(self):
```

Comment:
These two look like dupes.

Reply:
No These are not dupes , each has different purpose 
1) One is executing delete daemonset pipeline
2) Second is saving delete statefulset pipeline

Reply:
Using citest framework we are creating/deleting statefulset/daemonset 
Step 1) test_e_1_save_statefulset_pipeline/test_f_1_save_daemonset_pipeline 
        Here we are just creating a pipelines in spinnaker,it will be used to create statefulset/daemonset in kuberneties (Only if this is executed)
Step 2) test_e_2_create_statefulset_pipeline/test_f_2_create_daemonset_pipeline
        Here we are execution those pipeline we have just created above , so that it will create statefulset/daemonset in kubernetes.	
Step 3) test_g_1_save_delete_statefulset_pipeline/test_g_1_save_delete_daemonset_pipeline		
        Here we are just creating a pipelines in spinnaker to delete statefulset/daemonset ,we have just created. It will not delete until its been excuted
Step 4) test_g_2_execute_delete_statefulset_pipeline/test_g_2_execute_delete_daemonset_pipeline
        Here we are executing those pipeline we have just created above from citest , so that it will delete statefulset/daemonset what we just created in kubernetes.

Same ways you are creating test_d_create_find_image_pipeline and then running test_e_run_find_image_pipeline